base.mako

```diff
@@ -68,5 +68,35 @@
             ga('create', 'UA-104682927-1', 'auto');
             ga('send', 'pageview');
         </script>
+        <script>
+(function () {
+  function copyTextToClipboard(text) {
+    var textArea = document.createElement("textarea");
+    textArea.value = text;
+    textArea.style.top = "0";
+    textArea.style.left = "0";
+    textArea.style.position = "fixed";
```

Comment:
is it necessary to style this? could we use css here -- could this be placed offscreen (I believe this blinks in and out right now if I'm reading this correctly)

Reply:
- style is used to prevent scrolling when the element is added to the dom, but as I am testing it now, only `position` seem to be needed
- it does not work with css being defined somewhere else. My understanding for the reasoning is that this piece of script does not know about styles defined outside, and if it is desired to be used, one would have to import it somehow.
- right, it blinks for a tiny moment, but after using `position` only (as I tried it) it won't.

Reply:
`position: absolute; left: -9999px; top: -9999px;` might also be something to consider?

Reply:
- unfortunately, any position other than `fixed` will affect the scrolling, so I think that there is no run away from using it
- I used a negative number for `top` and `left` though, so that the textarea is created off-screen.